src/Integration/Binding/SolutionBindingOperation.cs

```diff
@@ -171,13 +171,16 @@ public void Prepare(CancellationToken token)
                     return;
                 }
 
-                var languageForProject = projectToLanguageMapper.GetAllBindingLanguagesForProject(project).First();
-                var bindingConfigFile = GetBindingConfig(languageForProject);
```

Comment:
The `languageForProject` and `bindingConfigFile` are only required if there is a project-level binder, so I moved those calls inside the new `if`.